auslib/db.py

```diff
@@ -1649,7 +1670,7 @@ def getRawMatches():
                 self.log.debug("%s doesn't match %s", rule['osVersion'], updateQuery['osVersion'])
                 continue
             # Same deal for system capabilities
-            if not self._csvMatchesRule(rule['systemCapabilities'], updateQuery.get('systemCapabilities', ""), substring=False):
```

Comment:
If this ends up making the substring mode of _csvMatchesRule unused, please remove support for it. Dead code--!